tests/unit/test_exceptions.py

```diff
@@ -79,6 +79,12 @@ def testCodeInvariants(self):
 
     def testInstantiation(self):
         for class_ in self._getExceptionClasses():
+            # some exceptions are becoming too complicated to instantiate
+            # like the rest of the exceptions; just do them manually
+            if class_ in [
+                    exceptions.RequestValidationFailureException,
+                    exceptions.ResponseValidationFailureException]:
+                continue
```

Comment:
Well, what about just creating a different args list for these classes and carrying on with the test?